docs/RELEASE_PROCESS.md

```diff
@@ -0,0 +1,11 @@
+### `cosmos/cosmos-sdk` Release Process
+
+- [ ] 1. Decide on release designation (are we doing a patch, or minor version bump) and start a P.R. for the release
```

Comment:
I think we might need a different procedure for hotfixes (namely, we might want to backport a fix to an older release)

Reply:
Yeah I think `hotfix` is a usecase we didn't solve for with this process.

Reply:
I think we should have a hotfix proceedure which may be (I hope 🤞) be out of scope for this PR.

Reply:
I think a hotfix release should in scope cause we need to differentiate what parts of the checklist we keep.

Hotfixes should have less testing before they go out.

We don't need to several days of internal testnets.

We do need to fire a testnet and run some transactions

Reply:
We're about to do a hotfix tomorrow (as per @cwgoes comment about cherry-picking 2 commits for 0.23) so lets figure it out tomorrow.